wpilibc/src/main/native/include/frc/apriltag/AprilTag.h

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#pragma once
+
+#include <wpi/SymbolExports.h>
+
+#include "frc/geometry/Pose3d.h"
+
+namespace frc {
+struct WPILIB_DLLEXPORT AprilTag {
+  int id;
+
+  Pose3d pose;
+
+  /**
+   * Checks equality between this AprilTag and another object.
+   *
+   * @param other The other object.
+   * @return Whether the two objects are equal.
+   */
+  bool operator==(const AprilTag& other) const;
+
+  /**
+   * Checks inequality between this AprilTag and another object.
+   *
+   * @param other The other object.
+   * @return Whether the two objects are not equal.
+   */
+  bool operator!=(const AprilTag& other) const;
```

Comment:
I don't think these need to be specified. Default operator== and operator!= should already exist for an aggregate type like this.